src/app/workout/plans/form.tsx

```
@@ -0,0 +1,102 @@
+                component="div"
+                variant="h2"
+            >
+                Create Workout Plan
+            </Typography>
+            <FormControl>
+                <InputLabel htmlFor="plan-name-input" sx={{fontSize: '25px!important'}}>Plan Name</InputLabel>
+                <Input
+                    id="plan-name-input"
+                    aria-describedby="plan-name-input-text"
+                    value={values?.name}
+                    onChange={(event: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)